libs/libzauth/libzauth-c/Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
should it be under vc, though?

Reply:
It helps to have .lock files under version control to always get the same thing when compiling today or in 6 months. 
If we were working on rust code a lot, it could make sense to not put it under version control, to notice when new versions of the rust compiler or some libraries become available and something becomes incompatible.